pkgs/by-name/zp/zpix-pixel-font/package.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "zpix-pixel-font";
+  version = "3.1.8";
+
+  srcs = [
+    (fetchurl {
+      name = "zpix-pixel-font.bdf";
+      url = "https://github.com/SolidZORO/zpix-pixel-font/releases/download/v${version}/zpix.bdf";
+      hash = "sha256-qE6YPKuk1FRRrTvmy4YIDuxRfslma264piUDj1FWtk4=";
+    })
+    (fetchurl {
+      name = "zpix-pixel-font.ttf";
+      url = "https://github.com/SolidZORO/zpix-pixel-font/releases/download/v${version}/zpix.ttf";
+      hash = "sha256-UIgLGsVTbyhYMKfTYiA+MZmV4dFT9HX3sxTdrcc4vE0=";
+    })
+  ];
+
+  dontUnpack = true;
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm444 $srcs/*.bdf $out/share/fonts/misc/zpix.bdf
+    install -Dm444 $srcs/*.ttf $out/share/fonts/truetype/zpix.ttf
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "A pixel font supporting multiple languages like English, Chinese and Japanese";
+    homepage = "https://github.com/SolidZORO/zpix-pixel-font/";
+    changelog = "https://github.com/SolidZORO/zpix-pixel-font/blob/master/CHANGELOG.md";
+    license = licenses.unfreeRedistributable;
```

Comment:
This license is [disputable](https://github.com/SolidZORO/zpix-pixel-font/#license-and-pricing). Software is by default unfree, and redistribution is not explicitly granted.

Reply:
The repo defines it free for non commercial usage, but i guess that does not explicitly defines it as fine for redistribution. 
It is also a really small precompiled font, so it probably does not really matter if it does not end up in the cache.